Packs/Gamma/pack_metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "Gamma",
+    "description": "Streamline discovery, classification and remediation of data loss instances across Enterprise SaaS applications (Slack, Jira, Github, Gdrive, Gmail, O365, etc.). Playbook engages with users while simultaneously allowing the Security team to instantly investigate and/or automatically remediate.",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Gamma Engineering",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "developer@gammanet.com",
+    "created": "2020-12-23T20:46:43Z",
+    "categories": [
+        "Network Security"
+    ],
+    "tags": [
+        "Security", "Compliance", "GDPR", "HIPAA", "PII", "IT", "Machine Learning"
+    ],
+    "useCases": [
+        "Compliance", "Employee Offboarding", "GDPR Breach Notification",
+        "HIPAA Breach Notification", "Hunting"
+    ],
+    "keywords": [
+        "Gamma", "DLP", "Data loss", "Data Discovery", "Data Exposure","Data Classification", "Slack",
+        "Github", "Gmail", "GDrive", "GSuite", "OneDrive", "Outlook", "O365", "Jira", "Confluence", "ServiceNow",
+        "zendesk", "SaaS", "Compliance", "GDPR", "HIPAA", "PII", "Machine Learning", "User Training"
+    ],
+    "githubUser": []
```

Comment:
It is highly recommended for partners to add a github user to get notifications whenever the pack changes.

Reply:
done, thanks